DemoApp/StreamChat/Components/DemoChatChannelListRouter.swift

```diff
@@ -518,6 +518,14 @@ final class DemoChatChannelListRouter: ChatChannelListRouter {
                         self?.showChannel(for: cid, at: message?.id)
                     }
                 }
+            }),
+            .init(title: "Delete Downloaded Attachments", handler: { _ in
+                guard FileManager.default.fileExists(atPath: URL.streamAttachmentDownloadsDirectory.path) else { return }
+                do {
+                    try FileManager.default.removeItem(at: .streamAttachmentDownloadsDirectory)
+                } catch {
+                    log.debug("Failed to delete downloaded attachments with error: \(error.localizedDescription)")
+                }
```

Comment:
Question: How to expose the way of deleting downloaded files?

Currently we just expose the path to the directory but that is pretty primitive. Considering adding a deleteAttachment method and deleteAllAttachments.

Reply:
Where do you plan to expose these methods?
For the `deleteAllAttachments` case - this path still works, so maybe we just keep the example here. Having it per attachment could be just a matter of exposing its local path, or you had sth else in mind?
Btw, we should mention "local", since this won't delete the attachment (`deleteAllLocalAttachments`).

Reply:
`attachment.downloadingState?.localFileURL` gives access to the local URL and one could use it to delete the local download. Downside is that the state in the DB is then incorrect. Therefore, I was thinking about separate methods. All depends how common it is to delete these files. Well, this reminds me that local attachments should be deleted on logout, otherwise they keep taking the space.

Reply:
Implemented the proposal above. Much better.